docs/reference/content/driver-async/getting-started/quick-start.md

```diff
@@ -13,6 +13,13 @@ title = "Quick Start"
 The following code snippets come from the [`QuickTour.java`]({{< srcref "driver-async/src/examples/tour/QuickTour.java">}}) example code
 that can be found with the async driver source on github.
 
+{{% note %}}
+There are two higher level MongoDB Asynchronous Java Drivers available, that users may find easier to work with:
```

Comment:
Let's expand this a bit to say _why_ users may find it easier to work with.  Given them a reason to click through.